bukkit/src/main/java/haveric/stackableItems/util/InventoryUtil.java

```diff
@@ -985,17 +985,11 @@ public static int getInventoryMax(Player player, String worldName, InventoryView
             gamemode = player.getGameMode();
         }
 
-        String invName = "";
-        if (view != null) {
-            invName = view.getTitle();
-        }
-
-
-        if (inventoryType == InventoryType.CHEST && (invName.equalsIgnoreCase("Horse") || invName.equalsIgnoreCase("Undead horse") || invName.equalsIgnoreCase("Skeleton horse"))) {
+        if (inventory.getType() == InventoryType.CHEST && StringUtil.equalsAnyIgnoreCase(view.getTitle(), "Horse", "Undead horse", "Skeleton horse")) {
```

Comment:
I'm pretty sure the view != null check is still necessary here. Possibly for virtual inventories. I'd have to check git history to verify.